Xamarin.Forms.Core/Markup/BindableObjectExtensions.cs

```diff
@@ -0,0 +1,183 @@
+		{
+			var converter = new FuncConverter<TSource, TDest, TParam>(convert, convertBack);
+			bindable.SetBinding(
+				targetProperty,
+				new Binding(path, mode, converter, converterParameter, stringFormat, source)
+				{
+					TargetNullValue = targetNullValue,
+					FallbackValue = fallbackValue
+				});
+			return bindable;
+		}
+
+		/// <summary>Bind to the default property</summary>
+		public static TBindable Bind<TBindable>(
+			this TBindable bindable,
+			string path = bindingContextPath,
+			BindingMode mode = BindingMode.Default,
+			IValueConverter converter = null,
+			object converterParameter = null,
+			string stringFormat = null,
+			object source = null,
+			object targetNullValue = null,
+			object fallbackValue = null
+		) where TBindable : BindableObject
+		{
+			bindable.Bind(
+				DefaultBindableProperties.GetFor(bindable),
+				path, mode, converter, converterParameter, stringFormat, source, targetNullValue, fallbackValue
+			);
+			return bindable;
+		}
+
+		/// <summary>Bind to the default property with inline conversion</summary>
+		public static TBindable Bind<TBindable, TSource, TDest>(
+			this TBindable bindable,
+			string path = bindingContextPath,
+			BindingMode mode = BindingMode.Default,
+			Func<TSource, TDest> convert = null,
+			Func<TDest, TSource> convertBack = null,
+			object converterParameter = null,
+			string stringFormat = null,
+			object source = null,
+			object targetNullValue = null,
+			object fallbackValue = null
+		) where TBindable : BindableObject
+		{
+			var converter = new FuncConverter<TSource, TDest, object>(convert, convertBack);
+			bindable.Bind(
+				DefaultBindableProperties.GetFor(bindable),
+				path, mode, converter, converterParameter, stringFormat, source, targetNullValue, fallbackValue
+			);
+			return bindable;
+		}
+
+		/// <summary>Bind to the default property with inline conversion and conversion parameter</summary>
+		public static TBindable Bind<TBindable, TSource, TParam, TDest>(
+			this TBindable bindable,
+			string path = bindingContextPath,
+			BindingMode mode = BindingMode.Default,
+			Func<TSource, TParam, TDest> convert = null,
+			Func<TDest, TParam, TSource> convertBack = null,
+			object converterParameter = null,
+			string stringFormat = null,
+			object source = null,
+			object targetNullValue = null,
+			object fallbackValue = null
+		) where TBindable : BindableObject
+		{
+			var converter = new FuncConverter<TSource, TDest, TParam>(convert, convertBack);
+			bindable.Bind(
+				DefaultBindableProperties.GetFor(bindable),
+				path, mode, converter, converterParameter, stringFormat, source, targetNullValue, fallbackValue
+			);
+			return bindable;
+		}
+
+		/// <summary>Bind to the <typeparamref name="TBindable"/>'s default Command and CommandParameter properties </summary>
+		/// <param name="parameterPath">If null, no binding is created for the CommandParameter property</param>
+		public static TBindable BindCommand<TBindable>(
```

Comment:
I don't think this should be exposed

Reply:
Could you clarify why?

This overload is just shorthand for two standard binding operations that are frequently used together:
- Bind a `CommandProperty`
- Bind the corresponding `CommandParameterProperty`

(typically in lists)
This overload does not expose anything new; it only combines two already public API calls